core-bundle/tests/Fixtures/Enum/IntBackedEnum.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Tests\Fixtures\Enum;
+
+enum IntBackedEnum: int
+{
+    case OptionA = 13;
+    case OptionB = 42;
```

Comment:
Just a nitpick, but our namig convention for enum cases is lowerCamelCase, see for example: https://github.com/contao/contao/blob/fd38b7fc2de60b418b835400087f42a3dc9de10c/core-bundle/src/Filesystem/SortMode.php

```suggestion
    case optionA = 13;
    case optionB = 42;
```

Reply:
Both enum fixtures and the test have been adjusted in https://github.com/contao/contao/pull/6584/commits/9b722df4d63e0628256195ef041f461222c5cb5b